<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1514</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="tabWidget_2">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1491</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QRadioButton" name="librarianRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Librarian</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="studentTeacherRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Student/Lecturer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>138</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="librarianLoginGroup">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>40</y>
        <width>651</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Librarian login</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="librarianPasswordLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>511</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="studentTeacherLoginGroup">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>150</y>
        <width>651</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Student/Lecturer Login</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="studentTeacherNameLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>511</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Name...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="studentTeacherPhoneLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>511</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Phone number...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="studentTeacherGmailLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>511</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>G-Mail address...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>GMAIL:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Add book</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addBookTitleLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>621</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Title...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addBookAuthorLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>621</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Author...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addBookISBNLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>621</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>ISBN...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ISBN:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addBookButton">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>180</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="addBookCopiesSpinBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>How many:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>711</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Remove book</string>
      </property>
      <widget class="QLineEdit" name="removeBookIdLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>581</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter Book ID to Remove...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Book ID:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeBookButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="librarianBooksTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>1491</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendUpdatesButton">
      <property name="geometry">
       <rect>
        <x>1320</x>
        <y>210</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Updates</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1511</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>User information</string>
      </property>
      <widget class="QLabel" name="studentTeacherNameDisplayLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>501</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="studentTeacherPhoneDisplayLabel">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>50</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
      <widget class="QLabel" name="studentTeacherGmailDisplayLabel">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>50</y>
         <width>571</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Gmail:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>771</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Book actions</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Borrow:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="borrowBookIdLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>531</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Book ID...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="borrowBookButton">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>40</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Borrow</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="returnBookIdLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>531</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter Book ID to Return...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="returnBookButton">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>80</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Return:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="reserveBookIdLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>531</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter Book ID to Reserve...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reserveBookButton">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>120</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Reserve</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Reserve:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cancelReservationIdLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>531</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter Book ID to Cancel Reservation...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelReservationButton">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>190</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel:</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="studentTeacherBooksTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>1501</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="studentTeacherIdDisplayLabel">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>140</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1514</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
